transformations/country_state_abbreviation_transformation/README.md

```diff
@@ -25,6 +25,30 @@ This transformation provides flexible options for Country/State abbreviation and
 | abbr  |  Boolean | Enable/disable full name -> abbreviation  |  True  | 
 | exp  | Boolean  | Enable/disable abbreviation -> full name | True  |  
 
+Benchmark results:
+
+- Sentiment analysis: 
+```
+python evaluate.py -t CountryStateAbbreviation -task "TEXT_CLASSIFICATION" -m "textattack/roberta-base-imdb" -d "imdb" -p 20
+```
+
+| Model  | Datasets | Performance  | 
+|---|---|---|
+| textattack/roberta-base-SST-2 | sst2 | 94.0->94.0 (0.0) | 
```

Comment:
Just to confirm, there was no difference in performance?

Reply:
Yes, for the classification tasks, there is no difference. For the text summarization task, the performance reduced on xsum dataset.